opaque.go

```diff
@@ -79,9 +79,9 @@ var (
 )
 
 type KSFConfiguration struct {
-	Identifier ksf.Identifier `json:"identifier"`
-	Parameters []int          `json:"parameters"`
-	Salt       []byte         `json:"salt"`
+	Identifier []int  `json:"identifier"`
```

Comment:
i believe that the author just wanted us to move the `Identifier` towards the bottom of this struct for memory alignment

Reply:
Ah, duh ahahaha should be fixed now ;)